src/client.js

```diff
@@ -33,21 +33,22 @@ var events = [
 	"whois"
 ];
 var inputs = [
+	// These inputs are sorted in order that is most likely to be used
+	"msg",
+	"whois",
+	"part",
 	"action",
+
 	"connect",
 	"invite",
 	"join",
 	"kick",
 	"mode",
-	"msg",
 	"nick",
 	"notice",
-	"part",
 	"quit",
 	"raw",
-	"services",
 	"topic",
-	"whois"
```

Comment:
This might be a stupid question, so don't be too tough on me :-)

Rather than making assumptions on most frequently used commands and calling them one by one until we have a match, why don't we directly math the input to the command name?
From what I see, `/join` matches to `join`, `/nick` matches to `nick`, ...
If this is not 100% the case, we can either forward commands, rename them, or treat the non-matching one differently.